package.json

```diff
@@ -49,14 +51,17 @@
     "popper.js": "1.12.9",
     "prop-types": "15.6.1",
     "react": "16.2.0",
+    "react-day-picker": "^7.0.5",
```

Comment:
Could you switch these to exact versions rather than tilde ranges? (and check each newly added dependency is specified at the latest version)